release notes/v1.2.2.md

```diff
@@ -0,0 +1,6 @@
+k6 1.2.2 is here 🎉! This is small patch release fixing a panic.
+
+## Bug fixes
+
+- [#5067](https://github.com/grafana/k6/pull/5067) fix: do not panic on `k6 login cloud`.
```

Comment:
```suggestion
- [#5067](https://github.com/grafana/k6/pull/5067) fixes a panic on the deprecated `k6 login cloud` command. Thanks @indygriffiths for reporting it!
```

* Keeps the same style as we do for other releases
* Thanks the user for reporting
* mention that the command is the deprecated one so they don't confuse it with the official command.